rsrc_issues.Rmd

```diff
@@ -1,38 +1,84 @@
+and [this one](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md) from [Noam Ross](https://ropensci.org/author/noam-ross/).
 
-Are you a package author who wants to encourage people to help with issues in your project? Consider making your packages “[contribution and collaboration friendly](https://devguide.ropensci.org/collaboration.html#friendlyfiles)” by including a [code of conduct](https://devguide.ropensci.org/collaboration.html#code-of-conduct), a contribution guide, and by labelling issues as suggested above. Some good examples of contribution guides are [this one](https://gist.github.com/peterdesmet/e90a1b0dc17af6c12daf6e8b2f044e7c) from [Peter Desmet](https://ropensci.org/author/peter-desmet/), and [this one](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md) from [Noam Ross](https://ropensci.org/author/noam-ross/).
+You can also label issues as suggested above. 
+We keep track of newly labelled `help wanted` issues and tweet about them to help
+developers get help.
 
 
 ### How to contribute? 
 
-*   For package authors: **Label your issues** so people interested in contributing can find them. We recommend “help wanted” (no hyphen), “good first issue”, “beginner”
+*   For package authors: **Label your issues** so people interested in contributing can find them. 
+    We recommend “help wanted” (no hyphen), “good first issue”, “beginner”
 *   For package authors: **Include future plans for your package** in the README and open issues to address aspects of your plan.
-*   **Address a “Help wanted” issue**. Take a look at the [Issues List](https://github.com/search?q=org%3Aropensci+label%3A%22help+wanted%22+state%3Aopen&type=Issues). If you see one that interests you, take a look at the project’s contributing guide, then comment in the issue to discuss your approach with the author. Create a [pull request](#channels-github) and submit your solution!
-*   **Address any open issue.** The majority of [issues](https://github.com/search?q=org%3Aropensci+state%3Aopen&type=Issues) are not labelled “help wanted” but that doesn’t mean the author wouldn’t appreciate your help. Browse the issues in a package you use and see if there’s one you could address. This is a great way to get your favorite packages work the way you want them to!
+*   **Address a “Help wanted” issue**. 
+    Take a look at the [Help Wanted page](https://ropensci.org/help-wanted/). 
+    If you see one that interests you, take a look at the project’s contributing guide, 
+    then comment in the issue to discuss your approach with the author. 
+    Create a [pull request](#channels-github) and submit your solution!
+*   **Address any open issue.** The majority of [issues](https://github.com/search?q=org%3Aropensci+state%3Aopen&type=Issues) are not labelled “help wanted” but that doesn’t mean the author wouldn’t appreciate your help. 
+    Browse the issues in a package you use and see if there’s one you could address. 
```

Comment:
```suggestion
    Browse the issues in a package you use and see if there's one you could address. 
```

Should there be a sentence about finding out about the repo's etiquette? E.g. issue before PR or not.